src/webhook_launcher/app/models.py

```diff
@@ -244,6 +244,11 @@ class WebHookMapping(models.Model):
         help_text="Choose Y to take content of revision as-is without "
                   "automatic processing (example: tarballs in git)",
     )
+    placeholder = models.BooleanField(
+        default=False,
+        editable=False,
+        help_text="Marks automatically created placehollders",
```

Comment:
"Marks automatically created placeholders. Intended as a filter for the admin UI. This flag should be cleared whenever the object is modified."

I wonder if there is value in checking the placeholder value during triggers? If so that should be mentioned in the text or a comment

Also double ll typo